app/models/challenge_signup.rb

```diff
@@ -102,70 +102,31 @@ def number_of_prompts
   # make sure that tags are unique across each group of prompts
   validate :unique_tags
   def unique_tags
-    if (challenge = collection.challenge)
-      errors_to_add = []
-      %w(prompts requests).each do |prompt_type|
-        restriction = case prompt_type
-          when "prompts"
-          then challenge.prompt_restriction
```

Comment:
`prompt_restriction` look unused. There was a [migration to rename it](https://github.com/otwcode/otwarchive/commit/53fdfda1d553b4d1bd3a628dc12dc1c4976e502e#diff-1459db5e5bb1426fc88b844f351ab93292c39ad14ddf80e2c7b8ccbb2bc0bf14), but the column is still around for some reason.

Not related to this issue, though.

Reply:
Huh, I knew that it was mostly unused, but I never realized that there was a migration that was supposed to have removed it.